hubapi/codelocations-api.go

```diff
@@ -28,6 +28,7 @@ type CodeLocation struct {
 	Name                 string     `json:"name"`
 	Type                 string     `json:"type"`
 	URL                  string     `json:"url"`
+	ScanSize             int32      `json:"scanSize"`
```

Comment:
Thank you for adding this item. Would you be kind to point me why `int32` is used, and not "just" `int`? "The sum of the size(s) of all scans associated with the code location" might be over 2mil of whatever is counted there.

Reply:
Changed it to int